src/drive/web/modules/upload/index.js

```diff
@@ -138,30 +173,55 @@ const uploadFile = async (client, file, dirID) => {
   return resp.data
 }
 
+export const getFilePath = async (client, file, dirID) => {
+  const resp = await client.collection('io.cozy.files').get(dirID)
+  const parentDirectory = resp.data
+  return `${parentDirectory.path}/${file.name}`
+}
+
+export const overwriteFile = async (client, file, path, dirID) => {
+  const statResp = await client.collection('io.cozy.files').statByPath(path)
+  const fileId = statResp.data.id
+  const resp = await client
+    .collection('io.cozy.files')
+    .updateFile(file, { dirId: dirID, fileId })
```

Comment:
I think we could use the `dirId` from `statResp `, no?

Reply:
:+1: